metisp/pymetis/src/pymetis/headers/headers.yaml

```diff
@@ -0,0 +1,33 @@
+headers:
+  DET.DIT:
+    name: "DET.DIT"
+    class: "header"
+    context: "template"
+    type: double
+    value: %.3f
+    unit: s
+    default: 1
+    range:
+      min: 0
+      max: 3600
+    description: "Detector integration time (average when NDIT > 1)"
```

Comment:
Please be aware that this particular format is based on the old "Data Interface Dictionary" format, which is being phased out. It is kinda unclear (to me at least) how it will be replaced.

So we should focus here on what is useful to us. E.g. `class` and `context` are not really useful. `value` isn't really either, but could be.

Reply:
For now I mostly focus on getting the structure right and I copied this mindlessly. This sound like less work for me.
A more pressing question for me is whether a YAML file is the correct place to put these things or not. So far I have not seen anything more complicated, but of course some relationships might be difficult to put into a file.